packages/terra-functional-testing/src/commands/utils/MemoryStream.js

```diff
@@ -0,0 +1,21 @@
+const stream = require('stream');
+
+/**
+ * MemoryStream is a PassThrough stream that keeps track of the length of the stream
+ */
+class MemoryStream extends stream.PassThrough {
+  constructor(options) {
+    super(options);
+
+    this.length = 0;
+  }
+
+  // eslint-disable-next-line no-underscore-dangle
```

Comment:
underscores are useful for internal or helper methods, do we want to disable this in the eslint.config instead of a single line disable?

Reply:
`_transforms` is a [steam API](https://nodejs.org/api/stream.html#transform_transformchunk-encoding-callback) so we're not able to modify the name. I think we should still keep this rule enabled since we typically don't recommend using the underscore in our code. This is a copy/paste from orion-releas-site.